Slim/Plugin/SpotifyLogi/Plugin.pm

```diff
@@ -86,7 +86,9 @@ sub initPlugin {
 	);
 	
 	# in LMS we're going to authenticate every player - don't do this on mysb, it's too expensive
-	Slim::Plugin::SpotifyLogi::ProtocolHandler->init();
```

Comment:
I don't know what exactly happened there. But maybe you should have used "git pull --rebase" to make sure your changes are applied on top of the latest code. Hard to tell...